starlark/unpack.go

```diff
@@ -89,9 +92,11 @@ func UnpackArgs(fnname string, args Tuple, kwargs []Tuple, pairs ...interface{})
 
 	paramName := func(x interface{}) string { // (no free variables)
 		name := x.(string)
-		if name[len(name)-1] == '?' {
-			name = name[:len(name)-1]
-		}
+
+		// name may end in "?" or "??"
+		name = strings.TrimSuffix(name, "?")
+		name = strings.TrimSuffix(name, "?")
```

Comment:
Change this function so that it returns (name string, skipNone bool), and avoid the redundant checks at each call.

Reply:
done